mlflow/models/evaluation/default_evaluator.py

```diff
@@ -260,15 +262,23 @@ def gen_line_x_y_label_fn(_y, _y_prob):
 
         xlabel = "False Positive Rate"
         ylabel = "True Positive Rate"
+        title = "ROC curve"
```

Comment:
Looks like `roc_curve` also has `pos_label`.

https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_curve.html

Reply:
Indeed. @jimmyxu-db I'll handle this in my PR